spec/tests/reverse_translator_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+require_relative '../spec_helper'
+
+RSpec.describe 'Openstudio::Metadata::ReverseTranslator spec' do
+  before(:all) do
+    building_type = 'SmallOffice'
```

Comment:
I think for this we should probably just include a `buildingtype_model.json` file with our expectations in there. Then I think we could actually provide more meaningful tests as to the success of the reverse translation.

Reply:
we could start off with small office, retail standalone, etc. users then could atleast look at the tests to see expectations for inferences that should be made based on what we have so far. or it could even be broken down into smaller chunks to start with (i.e. infer a unitaryheatpump, etc.). thoughts?

Reply:
I'm going to add an inputs directory to the specs because it think it would also be nice to have a place for templates and stuff as well outside of `lib/files`

Reply:
Toby, isn't this just duplicate data then? For things that are used specifically by the gem (mappings.json), I would rather not have duplicates. For things that are used to identify whether our current implementation matches expectations (i.e. smalloffice_model_haystack.json, smalloffice_model_brick.json, etc.) these are things I think should be added as new files.

Reply:
I guess my intention was to have mappings and templates that are locked in time, otherwise as we continue to modify the mappings and templates. In order to make sure that our `buildingtype_model.json` is accurate to the mappings and templates we're using we either need a locked in time copy or to generate the `buildingtype_model.json` on the fly everytime we run the test. I could go either way, but with locking a version in time to known valid values we can rule out mistakes in the mappings and templates in the `lib` directory as a reason for test failure.

Reply:
I feel like I would rather have the `buildingtype_model.json` file locked in time, with tests 'enforcing' as much as is known. as new mappings / templates get added, I would expect that our `buildingtype_model.json` gets updated, and new tests get added that 'prove' the new functionalities. thoughts? bad idea?

Reply:
I'm going to go ahead and move to this, I think your proposed approach is good for now. My goal with locking mappings and templates in time was to try to take them out of the equation as something that might be broken when the reverse translation spec fails, but I think this should be fine for now.